terraform/applications/careers/lambda-headers.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+// MUST be created in us-east-1
+exports.handler = (event, context, callback) => {
+   const request = event.Records[0].cf.request;
+   const response = event.Records[0].cf.response; 
+   const headers = response.headers; 
+   response.headers['x-frame-options'] = [{"key":"X-Frame-Options","value":"DENY"}]; 
+   response.headers['x-xss-protection'] = [{"key":"X-XSS-Protection","value":"1; mode=block"}]; 
```

Comment:
Is this script part of a template somewhere? There have been changes in web security standards since a few years ago when we were recommending XXP and I'd like to direct some comments in the right direction. (It's not relevant to *this* particular review, though.)

Reply:
I'm copying the script over from our existing deployment of careers, not sure if it was based on something or invented whole cloth, but I suppose here is the best place to have criticism of it, though maybe as a backlog item (changing plenty of things about careers, changing what headers we return might be asking for trouble).